packages/compat/plugin-webpack-swc/src/utils.ts

```diff
@@ -96,7 +96,16 @@ const isBeyondReact17 = async (cwd: string) => {
     return false;
   }
 
-  return isVersionBeyond17(deps.react);
+  try {
```

Comment:
Do we still need to read package.json?

![image](https://github.com/user-attachments/assets/36807245-991b-47c7-bfc1-3217b4750a60)

Reply:
I feel that it is necessary. The current project must have a React dependency. Otherwise, it is easy to use phantom dependencies.

Reply:
get👌